server/pages/index.js

```diff
@@ -145,7 +155,8 @@ function getDefaultContext( request ) {
 		isFluidWidth: !! config.isEnabled( 'fluid-width' ),
 		abTestHelper: !! config.isEnabled( 'dev/test-helper' ),
 		devDocsURL: '/devdocs',
-		store: createReduxStore( initialServerState )
+		store: createReduxStore( initialServerState ),
+		pride: pride,
```

Comment:
I wonder if we shouldn't pass an array of class names instead of specifying this specific one, so it's easier to add more in the future. But maybe it's premature optimization ...

As an aside, we can simplify this by using the [shortand syntax](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Object_initializer#Property_definitions):

```javascript
store: createReduxStore( initialServerState ),
pride,
```

Reply:
I don't mind adding that logic now, good suggestion.